visual-inspection/inspection-backend/inspection/model/model.py

```diff
@@ -30,10 +79,10 @@
     "mouse": "eine Maus",
     "magnetic_compass": "ein Kompass",
     "strainer": "ein Sieb",
-    "pill_bottle": "eine Pillendose",
+    "pill_bottle": "ein Tablettenfläschchen",
     "handkerchief": "ein Taschentuch",
     "saltshaker": "ein Salzstreuer",
-    "eggnog": "Eierlikör",
+    "eggnog": "ein Eierlikör",
```

Comment:
```suggestion
    "eggnog": "Eierlikör",
```

Makes more sense this way, I think.